### ISSUE TITLE
Unable to import requests

### ISSUE THREAD

**Issue by seanblundy** [meaningfy-ws/eds4jinja2#41]
`from eds4jinja2.builders.jinja_builder import build_eds_environment`

```
Exception has occurred: ModuleNotFoundError
No module named 'requests'
  File "C:\Users\User\Desktop\test.py", line 1, in <module>
    from eds4jinja2.builders.jinja_builder import build_eds_environment
```

https://github.com/meaningfy-ws/eds4jinja2/blob/0.1.33/eds4jinja2/adapters/prefix_cc_fetcher.py#L12

Import referenced in code but not in requirements.txt

**Comment by costezki** [comment]
thank you for reporting.

**Comment by costezki** [comment]
fixed in #42
